man/mtbl_fileset.3

```diff
@@ -60,6 +60,14 @@ mtbl_fileset_reload_now(struct mtbl_fileset *\fR\fB\fIf\fR\fR\fB);\fR
 mtbl_fileset_source(struct mtbl_fileset *\fR\fB\fIf\fR\fR\fB);\fR
 .fi
 .sp
+.nf
+\fBvoid
+mtbl_fileset_partition(struct mtbl_fileset *\fR\fB\fIf\fR\fR\fB,
+                mtbl_filename_filter_func \fR\fB\fIcb\fR\fR\fB,
+                struct mtbl_merger \fR\fB\fB\fIm1\fR\fR\fB,
+                struct mtbl_merger \fR\fR\fB\fIm2\fR\fR\fB);\fR
```

Comment:
The last two arguments are missing the `**` in the compiled version of the man page.

In the worst case, this could be fixed manually, committed, and noted in the commit.